app/models/ext_management_system.rb

```diff
@@ -50,6 +50,26 @@ def self.supported_for_catalog?
     catalog_types.present?
   end
 
+  def self.supported_for_label_mapping?
+    label_mapping_target_classes.present?
```

Comment:
Hey so what if we make this a `SupportsFeatureMixin` feature and the various providers can use `supports :label_mapping`?  Little bit more explicit and we can use it elsewhere e.g. UI/API where we're used to checking for supports features

Reply:
Done 👍